sport/app/rugby/conf/RugbyLifecycle.scala

```diff
@@ -20,28 +20,13 @@ class RugbyLifecycle(
   protected val initializationTimeout: FiniteDuration = 10.seconds
 
   override def start(): Unit = {
+    rugbyStatsJob.fetchFixturesAndResults()
+
     jobs.deschedule("FixturesAndResults")
-    jobs.schedule("FixturesAndResults", "5 0/30 * * * ?") {
+    jobs.schedule("FixturesAndResults", "30 * * * * ?") {
```

Comment:
Only run this once a minute, rather than every 5 seconds every two minutes before?